rtc/configuration/rtc-options-overview.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Overview of the Real-Time Collaboration configuration options
+title_nav: Configuration options overview
+description: List of all available RTC configuration options.
+keywords: rtc configuration
+---
+
+{% assign pluginname = "Real-Time Collaboration (RTC)" %}
+{% assign plugincode = "rtc" %}
+
+## Configuration style
+
+The Real-Time Collaboration (RTC) plugin uses promise-based "provider" functions to support a variety of configuration scenarios including asynchronously fetching data from a server. Function input parameters are provided as an object, allowing unused fields to be omitted.
+
+## Overview of RTC options
+
+### Required configuration options
+
+These options are _required_ when using the Real-Time Collaboration (`rtc`) plugin.
```

Comment:
"code" backticks on the word "rtc" is used here but not elsewhere

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/2091/commits/0c30d3f05b1fd7a4901fcdcd00832987e118e238